### PR TITLE
Added a check to check if a group affiliation data exists before getting uri and title. Fixes #81.

### LINKED ISSUE #81: Update code causing warning messages

### PR DESCRIPTION
**Pull request by sarongebre**
This will help remove the warning messages we see on CMS.

Also changed the way we are invalidating cache for sites that have groups.

### CODE REVIEW

File: unl_five.theme [hunk 1]
```diff
@@ -125,15 +125,8 @@ function unl_five_preprocess_page(&$variables) {
...
-
-    if ($group_storage) {
-      $group_ids = $group_storage->getQuery()
-        ->condition('type', 'group_subsite')
-        ->accessCheck(TRUE)
-        ->execute();
-
-      if ($group_ids) {
+    $group = \Drupal::entityTypeManager()->getStorage('group')->loadMultiple();
+      if ($group) {
```
Comment (ericras) [inline unl_five.theme]: Code formatting: indentation

File: unl_five.theme [hunk 2]
```diff
@@ -191,8 +183,10 @@ function unl_five_preprocess_block(&$variables) {
       $variables['site_name'] = $group->label();
       $variables['group_homepage_route'] = $string_url;
       $variables['group_id'] = $group_id;
-      $variables['affiliation_name'] = $group->get('group_subsite_group_affiliation')->getValue()[0]['title'];
-      $variables['affiliation_url'] = $group->get('group_subsite_group_affiliation')->getValue()[0]['uri'];
+      if($group->get('group_subsite_group_affiliation')->getValue()){
```
Comment (ericras) [inline unl_five.theme]: Code formatting: need space after "if" and before "{"

**Review by ericras — APPROVED** [review]

File: unl_five.theme (hunk 2 above)
Comment (sarongebre) [reply unl_five.theme]: Added the change

File: unl_five.theme (hunk 1 above)
Comment (sarongebre) [reply unl_five.theme]: Added the change

**Comment by ericras** [comment]
[8016972b4e20155170fe27a35ed43ae2ffb8e5fd](https://github.com/unlcms/project-herbie/commit/8016972b4e20155170fe27a35ed43ae2ffb8e5fd)

**Pull request merged**